website/src/components/Tasks/LabelTask/LabelTask.tsx

```diff
@@ -1,11 +1,13 @@
 import { Box, Button, Flex, HStack, Text, useColorModeValue } from "@chakra-ui/react";
-import { useEffect, useState } from "react";
+import { useEffect, useMemo, useState } from "react";
+import { useTranslation } from "react-i18next";
```

Comment:
Should it be `next-i18next`?

Reply:
correct, this was leftover code from some experiment so I removed it.